DEV-9/RandomReplacementPartsOfLines/Entrypoint.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+
+namespace RandomReplacementPartsOfLines
+{
+    class Entrypoint
+    {
+        static void Main(string[] args)
+        {
+            //Creating initial and replacing lines
+            string line = @"e:\TAT\GIT-1\DEV-9\RandomReplacementPartsOfLines\Lines.txt";
+            LinesReader linesReader = new LinesReader();
```

Comment:
there is no try- catch, initialLine and replacingLines can be null after file reading, exception is no not processed